CMakeLists.txt

```diff
@@ -16,14 +16,18 @@
 # To build with debugging use:
 # $ cmake .. -DCMAKE_BUILD_TYPE=Debug
 
-
-cmake_minimum_required (VERSION 2.6)
-
 ####################################################
 # Project
 ####################################################
+
+cmake_minimum_required(VERSION 2.6)
```

Comment:
We may want to raise this to a higher version. 2.6 is very old. I don't think it has support for some of the core modern CMake features, like targets.

Reply:
Go ahead and raise to an appropriate minimum version. This was not due to a requirement, just the example I used.